extensions/wikia/FeedsReportedPage/FeedsReportedPage.setup.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+$wgExtensionCredits['other'][] = array(
```

Comment:
What do you think of adding this functionality to the existing Feeds extension here as well?

Reply:
Same reason like for ucp, it was decided to use separate extension because it is more like a moderation tool, not only related to feeds itself.